### ISSUE TITLE
Improve “Support Center” Page Content & Structure

### ISSUE THREAD

**Issue by janavipandole** [janavipandole/Foodie#369]
Description: The current Support Center page provides basic support categories for users, but it lacks depth, accessibility enhancements, and consistency in layout. To improve user experience and reduce support friction, we propose a content and design update.

<img width="1500" height="975" alt="Image" src="https://github.com/user-attachments/assets/4b07e3db-e2b6-4baf-b76b-69e3535fb739" />

**Comment by janavipandole** [comment]
check that is ur change are visible or not if yes close this issue

**Comment by Nandan29300** [comment]
You close this issue @janavipandole
